src/eko/matching_conditions/n3lo/__init__.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+r"""
+This module defines the matching conditions for the N3LO |VFNS| evolution.
+
+The expressions are based on:
```

Comment:
Repeat this on all N3LO matching files

Reply:
I added it on purpose only in the `__init__`, I thought the spinx API would have been ugly

Reply:
Well, this is what we have to do ... the only other option would be not to repeat it in the docstring, but just as a comment

Reply:
I guess in `__init__` should be fine, if really really it's not enough, then simply add a comment to look to `__init__` for the full set of references.
I don't really want to blow up our code with copy-pasting of docstrings, if he will complain it's not my concern (everyone will consider it unreasonable, apart from him no one else care about citations in the code itself).

Reply:
No - we have to do it ... and indeed it will be *my* problem ;-) the files are big in any case, so let's just do it - as said also comments

Reply:
Nope, put a pointer, not the full copy-paste, it's unreasonable and we don't have to do anything anyone is going to ask us to do.

Even in papers you put references, and not the full bibliography every time you cite. Here we cite the block, and that's it.

Reply:
But this we had already - and it wasn't sufficient; the request was explicitly to put it everywhere

Reply:
It was in the code documentation, and nowhere in the code (so e.g. it was not distributed in the python package, since it only contains the code, and not the sphinx source). Now it's also in the code.

For everything else a pointer is sufficient, copy-paste does not add any value, people are able to open the file aside, and if they want to ignore, they would ignore anyhow.

Reply:
And in any case, don't be proactive in complaining in his place. If he'll complain, I'll reply as I'm replying to you, but don't fight for someone else, unless you genuinely agree with him.

Reply:
I do not agree, but I'm scared

Reply:
Okay, I'll add in each `OME`  docstring this sentence:
```
When using the code, please cite the complete list of reference, is available in :mod:`eko.matching_conditions.n3lo`
```
This sounds to me unambiguous and neat.
If you don't agree, I would go for repeat it as comment.